chart-pie.js

```diff
@@ -93,6 +97,37 @@ class ChartPie extends ResizeBehavior(ContextBehavior(ChartPropertyBehavior(mixi
       }]
     };
   }
+
+  // Source: https://stackoverflow.com/questions/33363373/how-to-display-pie-chart-data-values-of-each-slice-in-chart-js
+  _drawValuesInPie() {
+    const {ctx} = this.chart;
+    ctx.font = Chart.helpers.fontString(Chart.defaults.global.defaultFontFamily, 'normal', Chart.defaults.global.defaultFontFamily);
+    ctx.textAlign = 'center';
+    ctx.textBaseline = 'bottom';
+    ctx.fillStyle = '#fff';
+
+    this.data.datasets.forEach(function (dataset) {
```

Comment:
since we are using ecma 5, I think it would be more coherent if we would change this function to an arrow function
```suggestion
    this.data.datasets.forEach((dataset) => {
```